src/libs/OptionsListUtils.ts

```diff
@@ -1,13 +1,21 @@
 /* eslint-disable no-continue */
 import {parsePhoneNumber} from 'awesome-phonenumber';
 import Str from 'expensify-common/lib/str';
+// eslint-disable-next-line you-dont-need-lodash-underscore/get
 import lodashGet from 'lodash/get';
 import lodashOrderBy from 'lodash/orderBy';
 import lodashSet from 'lodash/set';
-import Onyx from 'react-native-onyx';
-import _ from 'underscore';
+import lodashSortBy from 'lodash/sortBy';
+import lodashTimes from 'lodash/times';
```

Comment:
We could try to use [native methods](https://lodash.com/docs/4.17.15#times), wdyt?